modules/patching-ovnk-address-ranges.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="patching-ovnk-address-ranges_{context}"]
+= Patching OVN-Kubernetes address ranges
+
+OVN-Kubernetes reserves the following IP address ranges:
+
+* `100.64.0.0/16`. This IP address range is used for the `internalJoinSubnet` parameter of OVN-Kubernetes by default. 
+* `100.88.0.0/16`. This IP address range is used for the `internalTransSwitchSubnet` parameter of OVN-Kubernetes by default.
+
+If OpenShift SDN has been configured to use either of these IP address ranges, you must patch them to use a different IP address range before initiating the limited live migration.
```

Comment:
If these IP addresses have been used by OpenShiftSDN and any external networks that may communicate with this cluster. Users shall use different CIDRs in OVN-K.